locomotor/__init__.py

```diff
@@ -452,10 +460,6 @@ def process_BoolOp(self, node, code, indent, loops):
     def process_Call(self, node, code, indent, loops):
         """Generate code for a function call"""
 
-        # We don't support positional or keyword arguments
-        if node.starargs or node.kwargs:
-            raise UntranslatableCodeException(node)
-
```

Comment:
Are you actually doing anything to address this? It doesn't seem to be handled anywhere.